test cases/frameworks/37 gir cpp/meson.build

```diff
@@ -0,0 +1,16 @@
+project('gnome module without C', 'cpp')
+
+glib = dependency('gobject-introspection-1.0', required: false)
+if not glib.found()
+  error('MESON_SKIP_TEST glib not found.')
```

Comment:
Should this say gobject-introspection not found?

Reply:
fixed